website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+
+> **A Data Engineer will document a dataset based on what they needed to know to build it. An Analyst with Analytics Engineering skills will document a dataset based on *how to use it going forward.***
+
+If we want to know how certain logic was built technically, then we can reference the SQL code in dbt docs. If we want to know *why* a certain logic was built into that specific model, then that’s where we’d turn to the documentation.
+
+- Example of not-so-helpful documentation ([dbt docs can](https://docs.getdbt.com/docs/building-a-dbt-project/documentation) build this dynamically):
+    - `Case when Zone = 1 and Level like 'A%' then 'True' else 'False' end as GroupB`
+- Example of better, more descriptive documentation (add to your dbt markdown file or column descriptions):
+    - Group B is defined as Users in Zone 1 with a Level beginning with the letter 'A'. These users are accessing our new add-on product that began in Beta in August 2022. It's recommended to filter them out of the main Active Users metric.
+
+In the second example, the documentation is written with the stakeholder and business in mind, and Analysts are experts at this translation of the technical to the non-technical.
+
+## Are you convinced?
+
+Teach your Analyst to be an Analytics Engineer, so they can do their own model building for curated datasets. You’ll start to see their process evolve into multiple smaller iterations, as they add fields and test them as they build. No loops, no hand-offs, and no Telephone Game. Your new workflow will look a little like this:
+
+![Comparing workflows with and without engineers](/img/blog/2022-09-28-analyst-to-ae/comparison_workflow.png)
+
+Instead of trying to define all the various groups of `Active Users` at onc, an Analytics Engineer can be validating the `Group A` rows, while adding `Group B` to their local environment, while still working with the stakeholder on the definition of `Group C`.
+
+**All this to say: Your Analyst turned Analytics Engineer is the key to unlocking fast development of curated datasets with next level data quality and documentation.**
+
+### How to empower your analysts to level-up their skillset
+
+Here’s some next steps to get you started on converting your analysts to analytics engineers:
+
+1. Many Analysts are already very comfortable querying data using SQL Select statements. For those that aren’t, challenge them to start pulling data for ad-hoc requests using SQL. Ask them to compare some of the common transformations in your BI tool to SQL functions or rebuild them using <Term id="cte">CTEs</Term>. This will prepare them for learning about dbt SQL models.
+1. Start incorporating peer review as part of the dashboard publishing process. Also, consider how you set-up your dashboard environments (do you have a local development area, a review area, and a published area?). This will prepare them for learning about Git, development environments, and version control.
```

Comment:
```suggestion
1. Start incorporating peer review as part of the dashboard publishing process. Also, consider how you set up your dashboard environments (do you have a local development area, a review area, and a published area?). This will prepare them for learning about Git, development environments, and version control.
```